client/src/components/v2/CameraRecording.module.css

```diff
@@ -0,0 +1,17 @@
+.button {
+  margin-right: 10px;
+}
+
+.footer {
+  display: flex;
+  align-items: center;
+  justify-content: center;
```

Comment:
Bootstrap buttons aren't usually centered. Personally I think it looks better aligned left

Reply:
![image](https://user-images.githubusercontent.com/50545432/89244935-13b8f080-d64b-11ea-8e30-57bb0a1657f8.png)
f49208f